### ISSUE TITLE
Chat not showing in games

### ISSUE THREAD

**Issue by ShadowFury15Q** [baffler/Transparent-Twitch-Chat-Overlay#332]
When i try launch the game chat not showing

**Comment by EDWernars** [comment]
True, but thats only when you using Fullscreen instead of borderless or windowed mode for a game.

so you can change that in the setting of your game if you need this.

**Comment by ShadowFury15Q** [comment]
Is it possible to somehow make the chat show in full-screen games?

**Comment by EDWernars** [comment]
It is possible, but you have to develop it. For now you can play in borderless.

**Comment by baffler** [comment]
There is a way but it requires a signed exe and basically running it like an accessibility app. Getting a signed exe isn't very easy or cheap, so it's not feasible for an open source project. There's other ways but it requires some advanced DirectX hooking and such and it's kind of out of the scope of this project.
